test/nodetool/test_info.py

```diff
@@ -98,7 +98,7 @@ def test_info(request, nodetool, display_all_tokens):
     expected_requests = [
         expected_request('GET', '/storage_service/gossiping', response=True),
         expected_request('GET', '/storage_service/hostid/local', response=host_id),
-        expected_request('GET', '/storage_service/rpc_server', response=True),
+        expected_request('GET', '/storage_service/rpc_server', response=False),
```

Comment:
Why don't we remove this REST API as well?

Reply:
as explained in another thread in this PR. it's preserved for backward compatibility. i could do this conditionally so that the native tool does not call this API, while keep the java-based tool happy. but i would leave this change to another PR.